server/controllers.js

```diff
@@ -0,0 +1,24 @@
+const db = require('./db/db-config');
+const rp = require('request-promise');
+
+
+// USERS -------------------------------->
```

Comment:
As this file grows, you may consider splitting these methods into different files by model.